html/js/utils.js

```diff
@@ -409,7 +409,7 @@ ut.setCropperModalImage = function (imgSrc) {
  * Retrieve the croped image.
  */
 ut.getCroppedModalImage = function () {
-    return $("#cropped_img").attr("src");
+    return $("#cropping_img").cropper("getCroppedCanvas", {width: 92, height: 92}).toDataURL();
```

Comment:
The magic number "92" is from the following CSS style.

    .image-circle-large {
        border-radius: 50%;
        width: 92px;
        height: 92px;
        border: 0px none;
    }